.github/workflows/android.yml

```diff
@@ -0,0 +1,23 @@
+name: Android CI
```

Comment:
I haven't tested this yet, but is based on the default skeleton provided by GitHub. Once merged, it should run the below actions for every pull request and push targeting develop.